test/unit/server/services/assessment.spec.js

```diff
@@ -0,0 +1,105 @@
+import sinon from 'sinon';
+
+import createAssessmentService from '../../../../server/services/assessment';
+
+describe('assessment service', () => {
+  let fakeDB;
+  let fakeAppInfo;
+  let assessment;
+  let result;
+
+  function setup() {
+    fakeDB = { raw: x => x };
+    fakeDB.insert = sinon.stub().returns(fakeDB);
+    fakeDB.into = sinon.stub().returns(fakeDB);
+    fakeDB.returning = sinon.stub().resolves([8979]);
+
+    fakeAppInfo = {
+      getGitRef: sinon.stub(),
+      getGitDate: sinon.stub(),
+      getQuestionHash: sinon.stub(),
+    };
+
+    assessment = createAssessmentService(fakeDB, fakeAppInfo);
+  }
+
+  describe('risk', () => {
+    describe('validation', () => {
+      it('should have some and be tested');
```

Comment:
No, but there's no downside? It was part of my original sketch, before I decided how to slice the implementation up into commits.

Will remove it from this commit though, easy enough either way.